src/test/java/org/jsoup/nodes/ElementTest.java

```diff
@@ -1319,6 +1319,16 @@ public void testNamespacedElements() {
         assertEquals("html > body > fb|comments", els.get(0).cssSelector());
     }
 
+    @Test
+    public void testSelectByNamespace() {
```

Comment:
This should be in SelectorTest, and needs to be more detailed -- e.g. to catch the example I pointed out, and validate other variations work.